emails/templates/emails/first_time_user.html

```diff
@@ -0,0 +1,411 @@
+
+                          <p style="line-height: 140%; text-align: left; word-wrap: break-word; color: #20123A">
+                            <strong>{% ftlmsg 'first-time-user-email-how-item-1-header' %}</strong> 
+                          </p>
+
+                          <p style="line-height: 140%; text-align: left; word-wrap: break-word;">
+                            {% ftlmsg 'first-time-user-email-how-item-1-subhead-html' url='https://addons.mozilla.org/firefox/addon/private-relay/?utm_campaign=relay-onboarding&utm_source=relay-onboarding&utm_medium=email&utm_content=addon-cta' %}
+                          </p>
+
+                      </td>
+                  </tr>
+
+                  <!-- second row -->
+                  <tr style="margin-bottom: 32px;display: inline-block;">
+                      <td style="padding:0 20px; vertical-align: middle;" align="center">
+
+                          <img align="center" border="0" src="{{ SITE_ORIGIN }}/static/images/email-images/first-time-user/privacy-icon.png" alt="" title="" style=" height: auto; width: 100%;max-width: 32px;" />
+
+                      </td>
+                      <td style="overflow-wrap:break-word;word-break:break-word;padding:10px 20px;font-family:inter;font-size:16px;" align="left">
+
+                          <p style="line-height: 140%; text-align: left; word-wrap: break-word; color: #20123A">
+                            <strong>{% ftlmsg 'first-time-user-email-how-item-2-header' %}</strong> 
+                          </p>
+
+                          <p style="line-height: 140%; text-align: left; word-wrap: break-word;">
+                            {% ftlmsg 'first-time-user-email-how-item-2-subhead' %}
+                          </p>
+
+                      </td>
+                  </tr>
+
+                  <!-- third row -->
+                  <tr style="margin-bottom: 32px;display: inline-block;">
+                      <td style="padding:0 20px; vertical-align: middle;" align="center">
+
+                          <img align="center" border="0" src="{{ SITE_ORIGIN }}/static/images/email-images/first-time-user/highlights-icon.png" alt="" title="" style=" height: auto; width: 100%;max-width: 32px;" />
+
+                      </td>
+                      <td style="overflow-wrap:break-word;word-break:break-word;padding:10px 20px;font-family:inter;font-size:16px;" align="left">
+
+                          <p style="line-height: 140%; text-align: left; word-wrap: break-word; color: #20123A">
+                            <strong>{% ftlmsg 'first-time-user-email-how-item-3-header' %}</strong> 
+                          </p>
+
+                          <p style="line-height: 140%; text-align: left; word-wrap: break-word;">
+                            {% ftlmsg 'first-time-user-email-how-item-3-subhead-html' url=SITE_ORIGIN %}
```

Comment:
Nit: is the dashboard a better target than the front page for sign in?

```suggestion
                            {% ftlmsg 'first-time-user-email-how-item-3-subhead-html' url=SITE_ORIGIN|add:'/accounts/profile/' %}
```